manager/src/main/java/com/redhat/service/bridge/manager/connectors/AbstractConnectorWorker.java

```diff
@@ -0,0 +1,89 @@
+package com.redhat.service.bridge.manager.connectors;
+
+import java.util.UUID;
+
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.transaction.Transactional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.bridge.manager.models.ConnectorEntity;
+
+import io.smallrye.mutiny.Uni;
+import io.smallrye.mutiny.tuples.Tuple2;
+
+public abstract class AbstractConnectorWorker<R> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractConnectorWorker.class);
+
+    @Inject
+    EntityManager entityManager;
+
+    // Could be the POD name
+    String workerId = UUID.randomUUID().toString();
```

Comment:
@lucamolteni Not sure this is going to work. I suspect it will just mean that we end up with orphaned items in the work queue. Imagine the following workflow:

- workerId that is random selects claims item
- workerId does first part of provisioning of connector
- workerId crashes

A new workerId is spawned by K8S. What happens to the item in the queue that was claimed by the previous worker? 

Using the Pod name as the `workerId` doesn't help as they are generated randomly via K8S controller when it creates them for the `Deployment` of the Fleet Manager.